client/state/account-recovery/reset/test/actions.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	fetchResetOptionsByLogin,
+	fetchResetOptionsByNameAndUrl,
+	fetchResetOptionsSuccess,
+	fetchResetOptionsError,
+} from '../actions';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+} from 'state/action-types';
+
+describe( '#fetchResetOptionsByLogin', () => {
+	it( 'should return ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST action with the user field', () => {
+		const user = 'foo';
+		const action = fetchResetOptionsByLogin( user );
+
+		assert.deepEqual( action, {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+			userData: {
+				user,
+			},
+		} );
```

Comment:
I can surely see how the BDD-style `expect` can be preferred in many cases. However, I personally find that it is not easy for me to memorize the proper order of the whole language chain, while I can write `assert` tests brainlessly. I think it might be that I am an ESL? ;)

If there is no strong preference of each, I would like to keep it as `assert` for now :)